docs/continuous_integration.rst

```diff
@@ -1,11 +1,44 @@
 Continuous Integration
 ======================
 
+The "CI" in CumulusCI stands for "continuous integration." Continuous integration is the practice of automatically running a project's tests for any change before that change is merged to the ``main`` branch in the repository. Continuous integration also configures the repository so that changes are merged only if the tests have passed. This practice keeps the ``main`` branch in an error-free state where it can be released any time.
+
+Teams can put in significant effort to create bespoke automation for CumulusCI tailored to their project's needs. When created, the automation is available to all project participants, from Developers and Quality Engineers, to Documentation Writers and Product Managers. CumulusCI takes this reuse of automation one step further by letting it run in the context of CI systems like GitHub Actions, CircleCI, or Azure Pipelines. This consistent reuse of automation from local environments to cloud-based CI systems gives teams the ability to develop, test, and deploy their projects with confidence.
```

Comment:
"From Developers and Quality Engineers, to Documentation Writers and Product Managers"

Is it standard to write all job titles in caps?

Reply:
Not sure, but I doubt it.